pkgs/applications/editors/vim/plugins/build-vim-plugin.nix

```
@@ -16,8 +16,8 @@ rec {
     , namePrefix ? "vimplugin-"
     , src
     , unpackPhase ? ""
-    , configurePhase ? ""
-    , buildPhase ? ""
+    , configurePhase ? ":"
+    , buildPhase ? ":"
```

would this be better?
```suggestion
    , configurePhase ? ""
    , buildPhase ? ""
```

---

changing this triggers more rebuilds, is there a diff in behavior ?

---

I don't think so, I just think this would be more intuitive

---

as I thought, these are not equivalent: `buildPhase ? ""` triggers the default builder so it will start building the default target of any Makefile, for instance aerial.nvim build will fail because it requires a whole new bunch of dependencies https://github.com/stevearc/aerial.nvim/blob/master/Makefile . I've put the default `buildPhase ? ":"` that skips the phase. It changes the behavior of buildVimPlugin but I dont think many use/need the current behavior. We can revisit before release if there are complaints.

---

also I've updated the changelog to notify of the breaking change

---

ooh good to know, then keeping this as `:` sounds good to me